call-notes/call_5.md

```diff
@@ -0,0 +1,152 @@
+
+**Maximilian Schneider:** No probably not. But I think you want to get back to normal operation as quickly as possible and I think that the delays probably within 100 slots for most applications. So that the time frame concerned are just very short.
+
+I wouldn't know what 4 hours ago another restart would change on the current situation right so of course unfortunately.
+
+**Zantetsu:** Okay thank you that does answer my question. Thank you.
+
+**Maximilian Schneider:** I had a follow-up for Alex, you mentioned that like the current hard Fork slot is currently synonymous for a restart but there's other ways to trigger a restart but there's no way there's no consensus stage shared about those resources right or is there another way to get those?
+
+**Alexander MeiBner:**  Yeah it's a kind of the other problem the security issue how are you gonna reach consensus about the restart slots right. But I was really talking just about the terminology that hard fork means that you are diverging on purpose because part of the cluster wants to do something else yeah
+and otherwise it's adjusted fork like any other fork that will get pruned at some point.
+
+**Maximilian Schneider:** Yeah it is a slow block right out basically so yeah I think the the differentiation between like a slow block and the restart is probably not so like on an application it doesn't really make a difference right if you have a one hour slow block or you have a hard fork it's probably the same right if it took like an hour to organize that or 24 hours.
+
+**Alexander MeiBner:** What I mean there's no like terminology called this or anything but.
+
+**Maximilian Schneider:** Yeah
+
+**Alexander MeiBner:** If people for example say like Bitcoin and Bitcoin Go these are hard folks, well like what they did on Ethereum back in the day. Because the community actually switched to an entirely different protocol version in the restart slot and this is not I mean this is obviously depends on the exact reset scenario but if you are just relearning the same version again always only minor block faces and all that we on the same thing then this is not really hard fork because it's not you're not forking off from anything else. But like what people understand on that term hard fork means that then there are two blocks and two sets of validators and to networks essentially which do a different phase. And I mean usually restart tries to avoid that scenario of hard forking and instead come back to one consensus of global networking.
+
+**Jacob Creech:** Alright interesting coming in some time because we are one minute overtime. Let's bring this discussion further into the SIMD. I will post it again in the chat and then we can further the discussion on this specifics SIMD 47 with Max and Alexander and more.
```

Comment:
```suggestion
**Jacob Creech:** Alright just being cognisant of time because we are one minute overtime. Let's bring this discussion further into the SIMD. I will post it again in the chat and then we can further the discussion on this specifics SIMD 47 with Max and Alexander and more.
```